server/etcdserver/txn/txn_test.go

```diff
@@ -35,6 +37,241 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestCheckTxn(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.TODO())
+	cancel()
```

Comment:
Move this into the sub test body and also `defer cancel()`?

Reply:
Done